src/main/resources/com/google/api/codegen/ruby/sample.snip

```diff
@@ -0,0 +1,128 @@
+@snippet generate(class)
+  @# BEFORE RUNNING:
+  @# ---------------
+  @# 1. If not already done, enable the {@class.apiTitle}
+  @#    and check the quota for your project at
+  @#    https://console.developers.google.com/apis/api/{@class.apiName}
+  @switch class.auth.type
+  @case "APPLICATION_DEFAULT_CREDENTIALS"
+    @# 2. This sample uses Application Default Credentials for authentication.
+    @#    If not already done, install the gcloud CLI from
+    @#    https://cloud.google.com/sdk and run
+    @#    `gcloud beta auth application-default login`
+    @# 3. Install the Ruby client library and Application Default Credentials
+    @#    library by running `gem install google-api-client` and
+    @#    `gem install googleauth`
```

Comment:
We might as well add the "more info" link for ADC (that was removed below) here now.